config/config.go

```diff
@@ -63,21 +64,117 @@ type DataStore struct {
 	DBNumber int
 }
 
+var defaultConfig = &config{
+	Munch: Munch{
+		Server: MunchServer{
+			Hostname: "localhost",
+			Port:     "50050",
+		},
+		LogLevel: "Logger",
```

Comment:
Shouldn't it be LogLevel: "info" (debug,info,warn,error,default - in logger.go)? The default will pick this up, but still..

Reply:
Yea it should be info i was just playing with the default and forgot to change it